cli.js

```diff
@@ -34,7 +34,10 @@ function start (opts) {
   }
 
   const stream = pinoElasticSearch(opts)
-
+    
+  stream.on('unknown', async (error) => {
```

Comment:
```suggestion
  stream.on('unknown', (error) => {
```

Why was this async?

Reply:
Sorry that was my mistake, to much async programming :-)